pages/cql/bcs-logic-2018.cql

```diff
@@ -64,72 +91,120 @@ define "Is Age 52 to 74 at End":
 	AgeInYearsAt(end of "Measurement Period") between 52 and 74
 
 define "Is In Hospice":
-	false
-	// TODO: Determine properly whether the patient was in hospice during the measurement year.
+	exists(
+		[Encounter: "Hospice Value Set"] Enc
+			where Enc.status.value in { 'planned', 'arrived', 'triaged', 'in-progress', 'onleave', 'finished' }
+				and Overlaps(PeriodToIntervalOfDT(Enc.period), "Measurement Period")
```

Comment:
Two comments here: 

1. We should not be using the prefix notation for the Overlaps invocation. Use the timing phrase. 
2. `X overlaps Y` is not the same as `X ends during Y`, so we should use the same notion as the original logic.